kola/harness.go

```diff
@@ -513,7 +513,7 @@ func getClusterSemver(flight platform.Flight, outputDir string) (*semver.Version
 		return nil, fmt.Errorf("creating new machine for semver check: %v", err)
 	}
 
-	out, stderr, err := m.SSH("grep ^VERSION_ID= /etc/os-release")
+	out, stderr, err := m.SSH("grep ^VERSION= /etc/os-release")
```

Comment:
I hope this won't break how the version of maintenance nightlies gets processed.